src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.h

```diff
@@ -2,18 +2,70 @@
 #define Z_EN_BIGPO_H
 
 #include "global.h"
+#include "../ovl_En_Clear_Tag/z_en_clear_tag.h"
 
 struct EnBigpo;
```

Comment:
```suggestion
#include "global.h"

struct EnBigpo;
```
One last comment! Because `Clear_Tag` is used by basically every enemy actor in the game, I put the params macro inside `z64actor.h` so this wouldn't need to be added for every file. This may or may not change in the future, but for now this isn't needed